integration/cluster_test.go

```diff
@@ -319,9 +320,12 @@ func TestIssue3699(t *testing.T) {
 	// node a will remain useless as long as d is the leader.
 	err := c.Members[0].Restart(t)
 	select {
+	// 5 seconds seems to be too brief
```

Comment:
this comment is confusing... probably just comment about why do we want to wait for 10 seconds instead?